mitodl/micromasters
# Capture user's full address

#### What are the relevant tickets?
Part of #2285. Adds two fields to the user profile form: street address, and postal code. The street address field is limited to 100 characters, a limitation that is reflected in the database backend. The postal code field only shows up if the country is "United States" or "Canada".

Note that the "state" field is present and required in all cases, even though Pearson only cares about it in some cases.

#### How should this be manually tested?
Play with these new fields. Make sure they accept valid input, and that they throw a validation error on invalid input (empty fields). Refresh the page, and make sure that the values are re-populated after being saved. Be sure that you can save an address outside of the United States and Canada without a postal code, and that you can't save an address within those two countries without a postal code.

#### Screenshots (if appropriate)
<img width="739" alt="screen shot 2017-01-10 at 3 07 22 pm" src="https://cloud.githubusercontent.com/assets/132355/21822821/c974cec4-d746-11e6-9e25-077167c299af.png">
<img width="738" alt="screen shot 2017-01-10 at 3 08 03 pm" src="https://cloud.githubusercontent.com/assets/132355/21822822/c9753e2c-d746-11e6-8351-0d2d2251f8f2.png">
